afterburner/src/main/java/com/fasterxml/jackson/module/afterburner/AfterburnerModule.java

```diff
@@ -2,14 +2,20 @@
 
 import com.fasterxml.jackson.core.Version;
 import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.util.NativeImageUtil;
 import com.fasterxml.jackson.module.afterburner.ser.SerializerModifier;
 import com.fasterxml.jackson.module.afterburner.deser.DeserializerModifier;
 
 public class AfterburnerModule extends Module
     implements java.io.Serializable // is this necessary?
 {
-    // TODO: replace with jackson-databind/NativeImageUtil.RUNNING_IN_SVM
-    private static final boolean RUNNING_IN_SVM = System.getProperty("org.graalvm.nativeimage.imagecode") != null;
+    /**
+     * <a href="https://github.com/FasterXML/jackson-modules-base/issues/191">[modules-base#191] Native image detection</a>
+     * 
+     * @since 2.16
+     */
+    private static final boolean RUNNING_IN_SVM = NativeImageUtil.isRunningInNativeImage();
```

Comment:
Not sure this can be done, since method Javadoc states:

```
     * This check cannot be a constant, because
     * the static initializer may run early during build time
```

Reply:
Yes, I think inlining would make sense: it will only be called once anyway.

Reply:
Not quite sure why logic uses part-static, part-dynamic logic there. I assume it's just an optimization to avoid dynamic check in some cases... but it really does seem odd. I think @yawkat might be able to shed more light on this.

Reply:
as discussed in #216, `isRunningInNativeImage()` should not be used here. There needs to be a new getter for `RUNNING_IN_SVM` that can be used instead.

Reply:
maybe `isRunningInNativeImage()` should be renamed also. i did not think enough about the name when i wrote it, since it wasn't public api. now that it is, a better name would be appropriate.

Reply:
Since the method was non-public before 2.16, it's fine to change the name I think.